cluster/manifests/prometheus/configmap.yaml

```diff
@@ -6,12 +6,61 @@ metadata:
   name: prometheus-conf
   namespace: kube-system
 data:
+  prometheus.rules.yml: |-
+    groups:
+    - name: skipper code by host
+      rules:
+      - record: job:skipper_code_by_host:sum
+        expr: sum(rate(skipper_serve_host_duration_seconds_count{application="skipper-ingress"}[1m])) by (code)
+
+    - name: skipper sum rate serve_host_duration by bucket
+      rules:
+      - record: job:skipper_serve_host_duration:sum
+        expr: sum(rate(skipper_serve_host_duration_seconds_bucket{application="skipper-ingress"}[1m])) by (le)
+
+    - name: skipper 2xx sum rate serve_host_duration by bucket
+      rules:
+      - record: job:skipper_serve_host_duration_2xx:sum
+        expr: sum(rate(skipper_serve_host_duration_seconds_bucket{code =~ "2.*",application="skipper-ingress"}[1m])) by (le)
+
+    - name: skipper sum rate filter_request_duration by filter
+      rules:
+      - record: job:skipper_req_filter_by_filter:sum
+        expr: sum(rate(skipper_filter_request_duration_seconds_bucket{application="skipper-ingress"}[1m])) by (le,filter)
+
+    - name: skipper sum rate filter_response_duration by filter
+      rules:
+      - record: job:skipper_resp_filter_by_filter:sum
+        expr: sum(rate(skipper_filter_response_duration_seconds_bucket{application="skipper-ingress"}[1m])) by (le,filter)
+
+    - name: skipper sum rate filter_request_duration by bucket
+      rules:
+      - record: job:skipper_req_filter_by_bucket:sum
+        expr: sum(rate(skipper_filter_request_duration_seconds_bucket{application="skipper-ingress"}[1m])) by (le)
+
+    - name: skipper sum rate filter_response_duration by bucket
+      rules:
+      - record: job:skipper_resp_filter_by_bucket:sum
+        expr: sum(rate(skipper_filter_response_duration_seconds_bucket{application="skipper-ingress"}[1m])) by (le)
+
+    - name: skipper sum rate custom totals by key
+      rules:
+      - record: job:skipper_custom_totals_by_key:sum
+        expr: sum(rate(skipper_custom_total{key !~ "^apiUsage.*", application="skipper-ingress"}[1m])) by (key)
+
+    - name: skipper sum rate average latency
+      rules:
+      - record: job:skipper_average_latency:sum
+        expr: sum(rate(skipper_serve_host_duration_seconds_sum{}[1m])) / sum(rate(skipper_serve_host_duration_seconds_count{}[1m]))
+
   prometheus.yml: |-
     global:
       scrape_interval: 15s
       external_labels:
         prometheus_replica: @@POD_NAME@@
         cluster: {{ .Cluster.Alias }}
+    rule_files:
+    - "prometheus.rules.yaml"
```

Comment:
This can be an absolute path apparently, so why not just have `/etc/prometheus/prometheus.rules.yaml` here and avoid the whole copy thing altogether?

Reply:
I jsut though there was a reason for doing it, so I did the same.
I am fine with changing it

Reply:
I think the only reason was replacing the pod name internally (don't remember why).

Reply:
changed as suggested